.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+dist: xenial
+sudo: false
+
+cache: pip
+
+install:
+  - pip install --upgrade pycodestyle
```

Comment:
Way overkill, but yes.
Again, for the sake of speed, we shoulsn't just now, since there are no tests as of yet.